WebInstall/explainer.md

```diff
@@ -14,15 +14,19 @@ Current paradigms of application discovery involve a user going to an applicatio
 
 While this is the general acquisition flow on many platforms, the web does not have the ability to replicate this scenario because it can not install applications. This makes it impossible for a web app (repository, catalog, store) to install and distribute other applications.
 
-**The Web Install API** addresses this shortcoming in the platform. It **allows a web site to install a web app. This functionality allows the creation of web based catalogs that can install PWAs directly from the web and into multiple platforms.**
+**The Web Install API** addresses this shortcoming in the platform. It **allows a web site to install a web app *(same or cross domain)*. This functionality allows the creation of web based catalogs that can install PWAs directly from the web and into multiple platforms.**
 
 ## Goals
 
-* **Enable installation of web apps (cross-domain).**
-* Enable UAs to supress potential installation-prompt spam. 
+* **Enable installation of web apps (same *and* cross-domain).**
+* Replace `beforeinstallprompt` or associated behaviour (current way to install apps from the same-domain).
+* Allow a ***vetted* installation origin** to know if the web app is installed (see *`install_sources` new manifest field*).
+* Allow the web app to report to the origin the outcome of the installation.
```

Comment:
WDYTA either 
```suggestion
* Allow the web app to report to the installation origin the outcome of the installation.
```

or (for readability, because I struggled with the original sentence):
```suggestion
* Allow the web app to report the outcome of the installation to the installation origin.
```

Then you're matching your use of "installation origin" above and specifying *which* origin: the store's or the web app's.